lib/file-helper.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const fs = require('fs')
+const nunjucks = require('nunjucks')
+
+// This helper function takes a path of a file and
+// returns the contents as string
+exports.getFileContents = path => {
+  let fileContents
+  try {
+    fileContents = fs.readFileSync(path)
+  } catch (err) {
+    if (err.code === 'ENOENT') {
+      console.log(err.message)
+    } else {
+      throw err
+    }
+  }
+  return fileContents.toString()
+}
+
+// This helper function takes a path of a nunjucks file and
+// returns the first macro code block inside that file
+exports.getRawMacro = path => {
```

Comment:
I think this function need renaming and its comments updating now that the examples don't include a macro?

Reply:
Yep, will update comment.